app/controllers/ontologies_controller.rb

```diff
@@ -52,6 +52,7 @@ def create
 
   def show
     @content_object = :ontology
+    params[:id] = params[:ontology_id] if params[:ontology_id]
```

Comment:
When I visit `http://ontohub.dev/default/partial_order`
My `params` hash is
```
{"controller"=>"ontologies", "action"=>"show", "repository_id"=>"default", "locid"=>"partial_order", "id"=>3}
```

(There is no `ontology_id`)